### PR TITLE
Fix Radio Button

### PR DESCRIPTION
**Pull request by singhj775**

### CODE REVIEW

File: service/radio_service.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def radio_action(adminVar):
```
Comment (sahityakmr) [inline service/radio_service.py]: Can we keep this method in mainframe_support.py?

File: service/radio_service.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    # selection = "You selected the option " + str(adminVar)
```
Comment (sahityakmr) [inline service/radio_service.py]: Please remove unused comments

File: ui/mainframe.py [hunk 3]
```diff
@@ -154,23 +154,23 @@ def __init__(self, mainUI=None):
         self.takeBackupBtn.configure(highlightcolor="black")
         self.takeBackupBtn.configure(pady="0")
         self.takeBackupBtn.configure(text='''Take Backup''')
-
-        userVar = tk.StringVar(mainUI, "User")
-        adminVar = tk.StringVar(mainUI, "Admin")
+        self.userVar = tk.StringVar(mainUI, "User")
```
Comment (sahityakmr) [inline ui/mainframe.py]: These variable shouldn't be a class variable so self is not needed here

Comment (sahityakmr) [reply ui/mainframe.py]: userVar is not required so please delete it.

File: ui/mainframe.py [hunk 4]
```diff
@@ -154,23 +154,23 @@ def __init__(self, mainUI=None):
...
+        self.adminVar = tk.StringVar(mainUI, "Admin")
```
Comment (sahityakmr) [inline ui/mainframe.py]: rename adminVar to some meaningful name becasuse we are using this for user and admin both.
We can use `userMode` as variable name

File: ui/mainframe.py [hunk 5]
```diff
@@ -154,23 +154,23 @@ def __init__(self, mainUI=None):
...
         self.style.map('TRadiobutton', background=[('selected', _bgcolor), ('active', _ana2color)])
         self.radioAdmin = ttk.Radiobutton(mainUI)
         self.radioAdmin.place(relx=0.65, rely=0.375, relwidth=0.091
                               , relheight=0.0, height=26)
         self.radioAdmin.configure(text='''Admin''')
+        self.radioAdmin.configure(variable=self.adminVar)
         self.radioAdmin.configure(value=1)
```
Comment (sahityakmr) [inline ui/mainframe.py]: Can this value be a String instead of a number?
If yes, please change it to a meaningful value like `user` and `admin`

**Review by sahityakmr — CHANGES_REQUESTED** [review]

File: service/radio_service.py (hunk 1 above)
Comment (singhj775) [reply service/radio_service.py]: yes

File: service/radio_service.py (hunk 2 above)
Comment (singhj775) [reply service/radio_service.py]: sure

File: ui/mainframe.py (hunk 3 above)
Comment (singhj775) [reply ui/mainframe.py]: ok

File: ui/mainframe.py (hunk 4 above)
Comment (singhj775) [reply ui/mainframe.py]: ok, how about radio_admin

Comment (singhj775) [reply ui/mainframe.py]: or about, user_type

Comment (sahityakmr) [reply ui/mainframe.py]: great

File: ui/mainframe.py (hunk 5 above)
Comment (singhj775) [reply ui/mainframe.py]: for admin value will be "admin"
use="user"

Comment (sahityakmr) [reply ui/mainframe.py]: perfect

Comment (singhj775) [reply ui/mainframe.py]: value changed but with name user_type_admin
user_type_user

File: service/radio_service.py (hunk 1 above)
Comment (sahityakmr) [reply service/radio_service.py]: this file is no longer needed, please delete it.

File: ui/mainframe.py [hunk 6]
```diff
@@ -154,23 +153,22 @@ def __init__(self, mainUI=None):
         self.takeBackupBtn.configure(highlightcolor="black")
         self.takeBackupBtn.configure(pady="0")
         self.takeBackupBtn.configure(text='''Take Backup''')
-
-        userVar = tk.StringVar(mainUI, "User")
         adminVar = tk.StringVar(mainUI, "Admin")
```
Comment (sahityakmr) [inline ui/mainframe.py]:
```suggestion
        logintype = tk.StringVar(mainUI, "Admin")
```

File: ui/mainframe_support.py [hunk 7]
```diff
@@ -74,6 +74,19 @@ def login(username, password):
         print("Login Failed")
 
 
+def radio_action(adminVar):
```
Comment (sahityakmr) [inline ui/mainframe_support.py]: Please do fomratting for this file,
Hit Ctr+Alt+L

**Review by sahityakmr — APPROVED** [review]

**Pull request merged**